common/src/main/java/com/wynntils/core/net/athena/WynntilsAccountManager.java

```diff
@@ -114,11 +116,22 @@ private void doLogin() {
                                 encodedConfigs.put(k.getKey(), k.getValue().getAsString()));
                 loggedIn = true;
                 WynntilsMod.info("Successfully connected to Athena!");
-                WynntilsMod.postEvent(new AthenaLoginEvent());
+
+                // It would be ideal to use an event here, but the event bus seems to be having issues with doing so.
```

Comment:
We have had events from thread pools before afaicr, so I don't think it's intrinsically impossible. But if you've spent enough time trying to get this to work, I'm okay with this solution.

Reply:
Yes, so I am really not sure why this doesn't work..